icu4j/main/core/src/test/java/com/ibm/icu/dev/test/TestBoilerplate.java

```diff
@@ -15,6 +15,7 @@
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
+import java.util.Map.Entry;
```

Comment:
Would you mind using Map.Entry in the code, rather than importing it?
I find that these nested-class imports generally lead to less readable call sites.
"What in the world is this Entry?"...

Reply:
I don't mind. That is also my preference.
I guess Eclipse added it and didn't notice it.

Done.